Dockerfile

```diff
@@ -1,22 +1,25 @@
 # This file is part of REANA.
-# Copyright (C) 2017, 2018, 2020 CERN.
+# Copyright (C) 2017, 2018, 2020, 2021 CERN.
 #
 # REANA is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-FROM ubuntu:16.04
+FROM rabbitmq:3.8.9-management
+
 # hadolint ignore=DL3009
-RUN apt-get update
-# hadolint ignore=DL3008
-RUN apt-get -y install --no-install-recommends rabbitmq-server
+RUN apt-get update && apt-get install -y --no-install-recommends curl=7.58.0-2ubuntu3.13 ca-certificates=20210119~18.04.1
 RUN apt-get -y autoremove && apt-get -y clean
+
 ENV RABBITMQ_NODENAME=rabbit@localhost
 ARG DEBUG=0
-RUN if [ "${DEBUG}" -gt 0 ]; then rabbitmq-plugins enable rabbitmq_management; fi;
-# hadolint ignore=DL3001
-RUN service rabbitmq-server start
+
+RUN curl -L "https://github.com/rabbitmq/rabbitmq-delayed-message-exchange/releases/download/3.8.9/rabbitmq_delayed_message_exchange-3.8.9-0199d11c.ez" > "$RABBITMQ_HOME/plugins/rabbitmq_delayed_message_exchange-3.8.9-0199d11c.ez"
+RUN chown rabbitmq:rabbitmq "$RABBITMQ_HOME/plugins/rabbitmq_delayed_message_exchange-3.8.9-0199d11c.ez"
+
+RUN rabbitmq-plugins enable --offline rabbitmq_delayed_message_exchange
+RUN rabbitmq-plugins enable --offline rabbitmq_consistent_hash_exchange
+RUN if [ "${DEBUG}" -lt 1 ]; then rabbitmq-plugins disable --offline rabbitmq_management; fi;
+
 COPY start.sh /start.sh
 RUN chmod 755 ./start.sh
-EXPOSE 5672
```

Comment:
I see 15672 is well exposed in the new base image... What about 5672?  We need to keep it in here, or else alter Helm port values?

Reply:
`15672` is exposed in [3.8-management](https://github.com/docker-library/rabbitmq/blob/7e63843da6bfb191ddee6dbe3dd7ec0df36ae70b/3.8/ubuntu/management/Dockerfile#L38) while
 `5672` is exposed in [rabbitmq:3.8](https://github.com/docker-library/rabbitmq/blob/d630cc389ee11f88a061a0c35e3aa7a928435293/3.8/ubuntu/Dockerfile#L276) which is a base image of `3.8-management`